src/components/BarChart.js

```diff
@@ -1,299 +1,351 @@
 const React = require('react');
-const numeral = require('numeral');
 const _isEqual = require('lodash/isEqual');
+const _merge = require('lodash/merge');
+const _omit = require('lodash/omit');
+const _functions = require('lodash/functions');
 
 const StyleConstants = require('../constants/Style');
 
-const SetIntervalMixin = {
-  componentWillMount () {
-    this.intervals = [];
-  },
-  setInterval () {
-    this.intervals.push(setInterval.apply(null, arguments));
-  },
-  componentWillUnmount () {
-    this.intervals.map(clearInterval);
-  }
-};
-
-const Rect = React.createClass({
+const Bar = React.createClass({
   propTypes: {
     animateOnHover: React.PropTypes.bool,
-    color: React.PropTypes.string,
-    height: React.PropTypes.number.isRequired,
-    label: React.PropTypes.string,
+    animationDuration: React.PropTypes.number,
+    height: React.PropTypes.number,
+    hovering: React.PropTypes.bool,
     onClick: React.PropTypes.func,
     onMouseOut: React.PropTypes.func,
     onMouseOver: React.PropTypes.func,
-    primaryColor: React.PropTypes.string,
-    value: React.PropTypes.number.isRequired,
+    radius: React.PropTypes.number,
+    value: React.PropTypes.number,
     width: React.PropTypes.number,
-    x: React.PropTypes.number.isRequired,
-    y: React.PropTypes.number.isRequired
-  },
-
-  mixins: [SetIntervalMixin],
```

Comment:
Mixins were the main thing preventing me from substituting React for Preact in the open source. I think if we could get rid of all the mixins we could probably substantially lower our bundle size with Preact.